src/cephadm/cephadm

```diff
@@ -1323,7 +1333,7 @@ def command_bootstrap():
     tmp_config = write_tmp(config, uid, gid)
 
     # a CLI helper to reduce our typing
-    def cli(cmd, extra_mounts={}):
+    def cli(cmd, extra_mounts={}, **kwargs):
```

Comment:
:-1: for adding `**kwargs`` here. If we need additional parameters, we should them list explictly.